code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -281,7 +281,7 @@
 			if(91.01 to INFINITY)
 				msg += "[t_He] [t_is] a shitfaced, slobbering wreck.\n"
 
-	if(has_trait(TRAIT_EMPATH) && !appears_dead && (src != user))
+	if(src != user && user.has_trait(TRAIT_EMPATH) && !appears_dead)

```

Comment:
Turns out has_trait is still a living proc around here, so I guess you'll have to check if the user is living beforehand et such, you know the drill.